.github/workflows/update-kvariants.yml

```diff
@@ -0,0 +1,29 @@
+name: Check for kVariants updates
```

Comment:
It does not look like a CI check to me, but more a CI that updates the dictionaries.
I would also add a comment to describe the CI, like "Create a PR update the dictionaries if a changes has been found" (for example)

Reply:
@curquiza Just pushed a [fixup commit](https://github.com/meilisearch/charabia/pull/192/commits/67a5e41af004dea31f1d2cb8315ad4b6ffd3ea49) that should address both of your remarks. 🙌